parallel-install/pkg/deployment/deletion.go

```diff
@@ -239,6 +242,68 @@ func (i *Deletion) deleteKymaNamespaces(namespaces []string) error {
 	finishedCh := make(chan bool)
 	errorCh := make(chan error)
 
+	// All the hacks below should be deleted after this issue is done: https://github.com/kyma-project/kyma/issues/11298
```

Comment:
would it be possible to move this hack-block into a method so that we can easier separate between the expected code and the workarounds?